Resources/Prototypes/Recipes/Construction/Graphs/weapons/makeshift laser gun.yml

```diff
@@ -0,0 +1,25 @@
+- type: constructionGraph
+  id: MakeshiftLaser
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: makeshiftLaser
+          steps:
+            - material: Cable
+              amount: 2
+              doAfter: 2
+            - material: Steel
+              amount: 10
+              doAfter: 4
+            - prototype: MaterialDiamond1
+              name: refined diamond
```

Comment:
The refined diamond prototype should be defined with everything else, for the sake of organisation and potential re-use in later recipes.